packages/webchat/src/main.tsx

```diff
@@ -120,11 +138,12 @@ class Web extends React.Component<MainProps> {
         return options
       }
     }
+    // TODO: 14 - parsing the config from the url is something that we don't need to do ourselves imho
```

Comment:
The shortlink /s/welcome-bot was used to redirect to the bot URL and configure various options, but I agree that we could move all the configuration in the messaging channel's config of the webchat.

Basically the studio received that URL then it loaded the webchat and displayed it to the users. We'll still need the reference tho, maybe other stuff also. we'll have to discuss

Reply:
I mean, the configuration of the channel in bot.config.json